Makefile

```diff
@@ -166,6 +166,10 @@ cephcsi: check-env
 e2e.test: check-env
 	go test $(GO_TAGS) -mod=vendor -c ./e2e
 
+.PHONY: rbd-group-snapshot
+rbd-group-snapshot:
+	go build -o _output/rbd-group-snapshot ./tools/rbd-group-snapshot
+
```

Comment:
@nixpanic can we revert this change in devel until we get agreement on having such tool in cephcsi?